FIDO2_messages_details.md

```diff
@@ -0,0 +1,188 @@
+## FIDO2 Authenticate Credential Request
+
+This message is sent as a request by the AUX MCU to authenticate a FIDO2 credential. The MAIN MCU responds
+with [FIDO2 Authenticate Credential Response](FIDO2_messages_details.md#fido2-authenticate-credential-response). The message is used to
+check whether the authenticator has the specified credential or not.
+
+**From AUX MCU:**
+
+| byte 0 - 251 | byte 252 - 283    |
```

Comment:
byte numbering to be changed I think

Reply:
Can you elaborate what you mean?

Reply:
exactly :). I'd prefer keeping byte number the same across all possible descriptions on this PR.  
It makes clear and not create any confusion